app/requestors/route.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class RequestorsRoute extends Route {
+  @service store;
+
+  async model() {
+    const requestors = await this.store.findAll('requestor');
+    return requestors;
```

Comment:
it's ok but that should be more clear 👌 
```suggestion
    return await this.store.findAll('requestor');
```